From 11364be98b200405630117d5eebb33bfdbc8c19e Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Sat, 7 Aug 2021 12:09:39 -0400
Subject: [PATCH 1/1] configure.ac: don't append -fabi-version=6 to CXXFLAGS.

This is a quick hack to avoid appending -fabi-version=6 to CXXFLAGS.
Both givaro and fflas-ffpack have problems with this legacy cruft
that are fixed upstream but tough to backport. We don't support
compilers that old, so we can just delete it.
---
 configure.ac | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index 571d923..616b2f5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -179,10 +179,6 @@ AC_SUBST(SIMD_FLAGS)
 dnl gcc-4.9.2 bug See https://trac.sagemath.org/ticket/17635#comment:178
 AS_IF([ test  "x$CCNAM" = "xgcc492" ],[REQUIRED_FLAGS="${REQUIRED_FLAGS} -fpermissive"],[])
 
-dnl With GCC's default ABI version, a __m128 or __m256 are the same types and therefore we cannot
-dnl have overloads for both types without linking error.
-AS_IF([test "x$CCNAM" = "xgcc"],[REQUIRED_FLAGS="${REQUIRED_FLAGS} -fabi-version=6"],[])
-
 AC_SUBST(REQUIRED_FLAGS)
 
 PARFLAGS="${OMPFLAGS}"
-- 
2.31.1

